### PR TITLE
implemented docker in project

### PR DESCRIPTION
**Pull request by SaiyaJ1N**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,9 @@
+MYSQL_ROOT_PASSWORD=RootPassword
+MYSQL_USER=admin
+MYSQL_PASSWORD=Password123
+MYSQL_DB=book_store
+MYSQL_LOCAL_PORT=3307
+MYSQL_DOCKER_PORT=3306
+SPRING_LOCAL_PORT=8087
+SPRING_DOCKER_PORT=8080
+DEBUG_PORT=5005
```
Comment (DmytroV95) [inline .env]: Please add empty line.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+# Final stage
+FROM openjdk:17-jdk-slim
+WORKDIR application
+COPY --from=builder application/dependencies/ ./
+COPY --from=builder application/spring-boot-loader/ ./
+COPY --from=builder application/snapshot-dependencies/ ./
+COPY --from=builder application/application/ ./
+ENTRYPOINT ["java", "org.springframework.boot.loader.JarLauncher"]
+EXPOSE 8080
```
Comment (DmytroV95) [inline Dockerfile]: Please add empty line.

File: docker-compose.yaml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    build: .
+    env_file: ./.env
+    environment:
+      SPRING_DATASOURCE_URL: jdbc:mysql://mysqldb:$MYSQL_DOCKER_PORT/$MYSQL_DB
+      SPRING_DATASOURCE_USERNAME: $MYSQL_USER
+      SPRING_DATASOURCE_PASSWORD: $MYSQL_PASSWORD
+      JAVA_TOOL_OPTIONS: -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:$DEBUG_PORT
+    ports:
+      - $SPRING_LOCAL_PORT:$SPRING_DOCKER_PORT
+      - $DEBUG_PORT:$DEBUG_PORT
```
Comment (DmytroV95) [inline docker-compose.yaml]: Please add empty line.

**Review by DmytroV95 — APPROVED** [review]
GJ. Left few comments. I was actually expecting to get a test assignment to review, but I got this one)). 

Also, I recommend you create env.sample with sample data and add original env. to .gitignore.

**Review by vovasalyha — APPROVED** [review]

**Pull request merged**